arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -167,42 +167,23 @@ public List<ReplayCompareResult> doContentCompare(ReplayActionCaseItem caseItem,
 
     List<ReplayCompareResult> replayCompareResults = new ArrayList<>();
     for (CategoryComparisonHolder bindHolder : waitCompareMap) {
-      if (operationConfig.checkIgnoreMockMessageType(bindHolder.getCategoryName(),
-          ignoreCategoryList)) {
+      if (operationConfig.checkIgnoreMockMessageType(bindHolder.getCategoryName())) {
         continue;
       }
-      filterIgnoredCompareItem(bindHolder, ignoreCategoryList);
       replayCompareResults.addAll(compareReplayResult(bindHolder, caseItem, operationConfig));
     }
-    return replayCompareResults;
-  }
-
-  private void filterIgnoredCompareItem(CategoryComparisonHolder bindHolder,
-      List<CategoryDetail> ignoreCategoryTypes) {
-    if (CollectionUtils.isEmpty(ignoreCategoryTypes)) {
-      return;
-    }
-    filterIgnoredCompareItem(bindHolder.getCategoryName(), bindHolder.getReplayResult(), ignoreCategoryTypes);
-    filterIgnoredCompareItem(bindHolder.getCategoryName(), bindHolder.getRecord(), ignoreCategoryTypes);
-  }
 
-  private void filterIgnoredCompareItem(String operationType, List<CompareItem> compareItems,
-      List<CategoryDetail> ignoreCategoryTypes) {
-    if (CollectionUtils.isEmpty(ignoreCategoryTypes)) {
-      return;
-    }
-    compareItems.removeIf(compareItem -> {
-      for (CategoryDetail categoryDetail : ignoreCategoryTypes) {
-        if (Objects.equals(categoryDetail.getOperationType(), operationType)
-            && (
-            categoryDetail.getOperationName() == null
-                || Objects.equals(categoryDetail.getOperationName(),
-                compareItem.getCompareOperation()))) {
-          return true;
-        }
+    //CategoryIgnore
+    replayCompareResults.forEach(compareResult -> {
```

Comment:
Is it feasible: 
Modify compareRecordAndResult and calculateMissResult methods or compareReplayResult to ignore before comparison

Reply:
done